src/lib/store/remote/helpers.js

```diff
@@ -106,13 +108,21 @@ exports.parseFromRemote = function(state, object) {
   // as in some cases IDs might contain '/', too
   //
   matches = id.match(/([^\/]+)\/(.*)/);
+  if (matches == null) {
+    console.log('hoodie: unhandled id ' + id);
```

Comment:
Ok, removed.  However, it's frustrating that warnings like this can't be logged somehow.  How about adding a hoodie function to log that can be injected for those who care?